.github/workflows/ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         uses: actions/checkout@v2
       - name: >
           Event: ${{ github.event_name }}
-          Repo: ${{ github.repository }}
+          Repo: ${{ steps.source-run-info.outputs.sourceHeadRepo }}
```

Comment:
This is an inconsistency I noticed - in the "CI Build" we've always showin "apache/airflow" here - which is the "target repo', but we should show the source repo (i.e. head Repo). This has no effect as we are not using job names to match the duplicates in CI.yml, but it's good for consistency.

Reply:
And it nicely works:

![Screenshot from 2020-11-01 14-12-00](https://user-images.githubusercontent.com/595491/97803908-92262e00-1c4c-11eb-9a7a-2ab52e248dca.png)

Reply:
It used to be like that:

![Screenshot from 2020-11-01 14-15-43](https://user-images.githubusercontent.com/595491/97803931-d9142380-1c4c-11eb-84a0-a2bb684fa4e4.png)